workout-view.php

```diff
@@ -1,14 +1,15 @@
 <?php 
     include('include/init.php');
     $programId = $_REQUEST['workoutId']; // switch to $programId
-    $workoutId = StartWorkoutFromProgram($programId);
 
     $program = GetProgram($programId,$_SESSION['userId']);
 
     $sets = GetSetsForWorkout($programId); //pulling from the original program
     $movements = GetMovementsForWorkout($programId);
 
     if(isset($_POST['complete'])){ //fill rows n stuff here
+
+        $workoutId = $_REQUEST['newWorkoutId'];
         FinishWorkout($workoutId);
```

Comment:
Hopefully FinishWorkout escapes the workoutid that's passed in directly from the request?